tests/Dockerfile

```diff
@@ -9,15 +9,31 @@ RUN curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | ba
   && nvm use default
 ENV PATH /root/.nvm/versions/node/v$NODE_VERSION/bin:$PATH
 # Check that Node.js was correctly installed
-RUN node --version 
+RUN node --version
+
+# Install Go
```

Comment:
Heck! That's a long installation process.
I wonder if we can trim this down some.

As I understand it https://github.com/moovweb/gvm is a bit more of a staple in the Go ecosystem than goenv. It may take considerably less to install. 🤔